### PR TITLE
Move tracks _ui and _ut properties out of filtered tracks properties

### LINKED ISSUE #33594: Tracks issue in some key store management actions
**Description:** we've identified a significant drop in the number of events fired when users interact with some key store management and setup actions:

`wcadmin_tasklist_task_completed`: significant drop (-80%) after the 6.6 release. It was also trending down after June 1 (maybe related to some a/b test issue).

`wcadmin_inbox_action_click`: significant drop (-80%) after the 6.6 release.

`wcadmin_orders_edit_status_change`: significant drop (-50%) after the 6.6 release.

`wcadmin_orders_view`: significant drop (-50%) after the 6.6 release.

`wcadmin_settings_view`: significant drop (-50%) after the 6.6 release.

`wcadmin_settings_change`: significant drop (-50%) after the 6.6 release.

`wcadmin_product_add_publish`: significant drop (-50%) after 6.4 and another drop (-70%) after 6.6.

`wcadmin_products_view`: significant drop (-50%) after the 6.6 release.

### PR DESCRIPTION
**Pull request by joshuatf**
### All Submissions:

-   [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md)?
-   [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
-   [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change?

### Changes proposed in this Pull Request:

Closes #33594 .

### How to test the changes in this Pull Request:

1. Open live tracks in MC (filter by your username if logged in).
2. Navigate to various WC pages to trigger some events
3. Note that expected events come through without being rejected

### Other information:

-   [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
-   [ ] Have you written new tests for your changes, as applicable?
-   [ ] Have you successfully run tests with your changes locally?
-   [ ] Have you created a changelog file for each project being changed, ie `pnpm changelog add --filter=<project>`?

### FOR PR REVIEWER ONLY:

-   [ ] I have reviewed that everything is sanitized/escaped appropriately for any SQL or XSS injection possibilities. I made sure Linting is not ignored or disabled.

### CODE REVIEW

**Comment by botwoo** [comment]
:bar_chart: Test reports for this pull request have been published and are accessible through the following links:

- [REST API Test Report](https://woocommerce.github.io/woocommerce-test-reports/pr/33621/api/)
- E2E Test Report (TBC)

Latest commit referenced in the reports: `Add changelog entry` 975fe079630b346e89632cfef895ddc1d31c2609
This comment will automatically be updated with the latest referenced commit when you push new changes to this pull request.

---

_Visit the [WooCommerce Test Reports](https://woocommerce.github.io/woocommerce-test-reports) homepage to view all published reports. See the [FAQs](https://github.com/woocommerce/woocommerce-test-reports/wiki/FAQs) page if you're having problems accessing them._

**Comment by AnnaMag** [comment]
Tested the functionality by looking at the composed pixel
`https://pixel.wp.com/t.gif?_en=wcadmin_product_edit&_ts=1656341611292&product_id=12&_via_ua=Mozilla%2F5.0+%28Macintosh%3B+Intel+Mac+OS+X+10.15%3B+rv%3A101.0%29+Gecko%2F20100101+Firefox%2F101.0&_via_ip=127.0.0.1&_lg=en-US%2Cen%3Bq%3D0.5&_dr=http%3A%2F%2Fbanana.local%2Fwp-admin%2Fpost.php%3Fpost%3D12%26action%3Dedit%26XDEBUG_SESSION_START%3D12716&_dl=&url=http%3A%2F%2Fbanana.local&blog_lang=en_US&products_count=2&wc_version=6.7.0&`**_ut**`=anon&`**_ui**`=woo%3ARP0wBf9lP5stFHcx6XVlcuzC&browser_type=php-agent`.

LGTM.

File: plugins/woocommerce/includes/tracks/class-wc-tracks.php [hunk 1]
```diff
@@ -38,20 +38,37 @@ public static function record_event( $event_name, $properties = array() ) {
...
-
-		// Allow event props to be filtered to enable adding site-wide props.
-		$filtered_properties = apply_filters( 'woocommerce_tracks_event_properties', $properties, $prefixed_event_name );
-
-		// Delete _ui and _ut protected properties.
-		unset( $filtered_properties['_ui'] );
-		unset( $filtered_properties['_ut'] );
-
-		$event_obj = new WC_Tracks_Event( $filtered_properties );
+		$event_properties    = self::get_properties( $event_name, $properties );
```
Comment (AnnaMag) [inline plugins/woocommerce/includes/tracks/class-wc-tracks.php]: `$properties` are not defined.

**Comment by AnnaMag** [comment]
Almost there! The latest refactor seems to break things a little.

**Comment by joshuatf** [comment]
Thanks, @AnnaMag!  Fixed and pushed.

**Review by AnnaMag — DISMISSED** [review]
Tests well. Thanks!

**Review by AnnaMag — APPROVED** [review]
LGTM

**Pull request merged**